app/api/resources/metadata.py

```diff
@@ -117,10 +116,12 @@ async def example_objects(request: Request, genome_id: str):
                 example_objects.model_dump()["example_objects"]
             )
         else:
-            not_found_response = {
-                "message": "Could not find example objects for {}".format(genome_id)
-            }
-            response_data = responses.JSONResponse(not_found_response, status_code=404)
+            raise HTTPException(
+                status_code=404,
+                detail=f"Could not find example objects for {genome_id}"
+            )
+    except HTTPException:
+        raise
```

Comment:
Does the `response_error_handler` mechanism not working.  IIRC `response_error_handler` was introduced to make it consistent across all error codes and the [error_response](https://github.com/Ensembl/ensembl-web-metadata-api/blob/fd0958f296ce01fe7b3caf55abd4efff8304f37a/app/api/error_response.py#L12) would be the single point of handling errors. 
The question is do we want to diverge from this ?

Reply:
Thanks for pointing this out, I used `response_error_handler` as you advised and made it customizable based on the error message